### PR TITLE
Feature: Display WebUI version on homepage

### LINKED ISSUE #94: Show WebUI version in the ui
Add WebUI version to version info to footer:

```
Version Info
Flexget: 3.1.22
Flexget UI: 2.012
API: 1.7.1
```

### PR DESCRIPTION
**Pull request by shadycuz**
### Motivation for changes:
It was a request and it was low hanging fruit 🥝 🍎 🍌 
### Detailed changes:
- Passed npm_package_version variable to webpack
- Display npm_package_version  in Version component
- Update Snapshot that failed

### Addressed issues:
- Fixes #94 

Not really sure if passing the npm package version to webpack is the correct way. Also not sure if this will work for a "production" build as I'm not familiar with webpack or how you bundle the webui with the flexget binary. 

image

### CODE REVIEW

**Comment by kristenmills** [comment]
Thanks. I'll take a look in a bit.

File: webpack.config.js [hunk 1]
```diff
@@ -33,7 +33,10 @@ if (__DEV__) {
 }
 
 const plugins = [
-  new webpack.DefinePlugin({ __DEV__ }),
+  new webpack.DefinePlugin({
+    __DEV__,
+    'process.env.npm_package_version': JSON.stringify(process.env.npm_package_version)
```
Comment (kristenmills) [inline webpack.config.js]:
```suggestion
    __VERSION__: JSON.stringify(process.env.npm_package_version),
```
you'll also have add this to the globals in the eslintrc https://github.com/Flexget/webui/blob/develop/.eslintrc.js#L30

**Comment by kristenmills** [comment]
Actually we might have to change this for production. We don't bump the version currently until after doing a production build. I can point you in the direction of all that stuff, are you in the flexget chat?

**Comment by shadycuz** [comment]
@kristenmills busy week at work. I'll be free all weekend. Ping me on gitter and we can talk about this feature.

**Comment by kristenmills** [comment]
Sorry that i haven't gotten to this yet @shadycuz I've been really busy myself

**Comment by shadycuz** [comment]
@kristenmills No problems. I have tried DMing you on gitter, but maybe I should try slack?

**Comment by kristenmills** [comment]
Yeah message me on slack.

**Pull request merged**